python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -74,6 +73,7 @@ class Node(object):
     thread-safe usage.
     """
     __metaclass__ = abc.ABCMeta
+    __slots__ = ('_index_cache_dict', '_parent', '_read_lock', '_write_lock', '_dirty', '_registry', '_data_dict')
```

Comment:
ah well... that's the advantage of forgetting the day to day code base and how it operates on the practical level, I can just pontificate about things that would be nice and a good idea. Looks like then it would be too much work and too intrusive to ban outright dynamic variables now :wink: